pom.xml

```diff
@@ -587,6 +587,9 @@
         <module>examples/spark2</module>
         <module>datamap/lucene</module>
         <module>datamap/bloom</module>
+        <!-- GCC ,C++,CMAKE,MAKE and TestFramework is added only for spark 2.2 related machines,So calling
```

Comment:
Why don't support spark2.3? or TODO

Reply:
2 point

Reply:
Actually CI machines are not updated with GCC and Cmake,Make,googletest.  SDK is  not related to any spark version .  i updated GCC,CMAKe,MAKE,googletest  only for spark2.2 machines.

Reply:
Can you support spark-2.3 too?

Reply:
SDK does not depend on Spark Version.  Currently it is added in spark2.2 profile because only spark2.2 related CI machine are having required software(make,cmake..etc).